src/main/java/banking/Main.java

```diff
@@ -0,0 +1,16 @@
+package banking;
+
+import banking.database.DBConnector;
+
+public class Main {
+    public static void main(String[] args) {
+        DBConnector connector = new DBConnector();
+
+        if (args.length > 0 && args[0].equals("-fileName")) {
+            connector.createDatabase(args[1]);
+        }
+        connector.createTable();
+        Menu menu = new Menu(connector.getUrl());
```

Comment:
What does "Menu" mean? Why does this class keep information about connector URL?

Reply:
fixed